components/BoostContentCard.tsx

```diff
@@ -137,16 +139,35 @@ const BoostContentCard = ({ content_txid, content_type, content_text, count, dif
                             </div>
                             )}
                             <div className='grow'/>
+                            
                             <a  onClick={(e:any)=>e.stopPropagation()}
                                 target="_blank"
                                 rel="noreferrer"
                                 href={`https://whatsonchain.com/tx/${content_txid}`}
                                 className="text-xs leading-5 whitespace-nowrap text-gray-500 dark:text-gray-300 hover:text-gray-700 hover:dark:text-gray-500"
+                                id="txid"
```

Comment:
an id is supposed to be a **unique** identifier in the DOM. You cannot have different elements with the same id named "txid".

I recommend using the actual txid as an id in that case :)

Reply:
Great catch!

Reply:
I implemented the id in the way you recommended but now I'm seeing that only some txids get the tooltip and not others, any ideas?

<img width="525" alt="Screen Shot 2023-02-28 at 1 02 28 AM" src="https://user-images.githubusercontent.com/19578049/221804344-6257248f-d899-49bc-9338-dacf77b85009.png">
<img width="1440" alt="Screen Shot 2023-02-28 at 1 02 15 AM" src="https://user-images.githubusercontent.com/19578049/221804348-af3b5464-5d84-4b97-b2df-73784b0f6da8.png">
<img width="1440" alt="Screen Shot 2023-02-28 at 1 02 08 AM" src="https://user-images.githubusercontent.com/19578049/221804354-da6caf2d-e426-4984-8f0d-3445bcf950a3.png">

Reply:
might be that `content_txid` does not exist on that specific post (could be named `txid` or `transaction`)

Reply:
fixed!